CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ add_library(devilution STATIC
   Source/pack.cpp
   Source/palette.cpp
   Source/path.cpp
+  Source/paths.cpp

```

Comment:
DevilutionX specific files should live in SourceX, Source things we inherited from Devilution (with minimal changes)

Reply:
This file is used in both Source and SourceX
We don't have a separate target for SourceS so it can't live there

What's the best option here?

Reply:
Is this what you had in mind?

https://github.com/diasurgical/devilutionX/pull/299/commits/4c3c5866277dd620b2a803329015c9c316cc2bee

Reply:
SourceS or more or less a temporary thing for headers that are used to avoid function naming conflicts on Windows.
Since paths.cpp is devilutionx code that replaces a handful of winapi functions originally used I would say that it belongs in SourceX, SourceS is acceptable for now though.